Automattic/pocket-casts-ios
# Feature Flag: add toggle screen

When developing, we often toggle Feature Flags on/off. So far, this has been done manually editing `FeatureFlag.swift`.

This PR adds a screen to toggle FeatureFlag values and persist them among builds/branches — the same way (and same code!) WPiOS do it.

We can also improve this screen with a better description of the Features but I wanted to keep this PR as simple as possible.

**This is only available for DEBUG builds.**

| New option under Settings | Beta Features screen |
| -------------------------- | --------------------- |
| ![Simulator Screen Shot - iPhone 14 - 2022-12-08 at 13 32 13](https://user-images.githubusercontent.com/7040243/206509033-98131069-a6c7-4c4e-a945-1cb3646b7eb9.png) | ![Simulator Screen Shot - iPhone 14 - 2022-12-08 at 10 53 16](https://user-images.githubusercontent.com/7040243/206509116-1a1b0b97-53e2-41f7-b812-b31a19823e70.png) |

## To test

1. Run the app
2. Go to Settings
3. Tap "Beta Features"
4. Toggle "End of Year" (off)
5. ✅ Go to Profile and check that the card disappeared
6. Go back to Settings > Beta Features > toggle End of Year on
7. ✅ Go back to Profile and check that the card is back
8. You can also check the other flags

## Checklist
